web/src/components/molecules/Asset/AssetList/AssetListTable/index.tsx

```diff
@@ -75,13 +74,13 @@ const AssetListTable: React.FC<AssetListTableProps> = ({
       title: "Created At",
       dataIndex: "createdAt",
       key: "createdAt",
-      sorter: (a, b) => dateSortCallback(a.createdAt, b.createdAt),
+      sorter: (a, b) => dateSortCallback(new Date(a.createdAt), new Date(b.createdAt)),
```

Comment:
I'm just curious, what scenario did you face to have this change?

Reply:
Done!
I added the `string | Date` type to the dateSortCallback function instead.